### DISCUSSION TITLE
Is it possible to attach remote duckdb database?

### DISCUSSION THREAD

**Discussion by kesavkolla** [duckdb/duckdb-wasm#1240] [Q&A]
I have a duckdb database on server side. Is it possible to fetch that database and attach it to duckdb in wasm inside browser?

**Comment by carlopi** [comment]
I would expect on this feature parity with DuckDB CLI, does `ATTACH 'https://remote.org/database/'` works on the CLI? If so the same should work at the SQL level (but might not be properly exposed to the JavaScript interface yet).

**Comment by kesavkolla** [comment]
In DuckDB CLI ATTACH '/path/test.db' works. That is basically reads the test.db from /path. In wasm there is no filesystem so wondering is there anyway to fetch the database and then attach to duckdb?

**Comment by kiyosoft** [comment]
Any update on this one?

**Comment by vellotis** [comment]
By chance saw this discussion and remembered I saw something related on the web.

It seems to be doing exactly you require.
https://observablehq.com/@huggingface/attach-a-duckdb-file-to-duckdb-wasm
